docs: roadmap update (#4852)
neondatabase/website: content/docs/introduction/roadmap.md

- **Neon Auth general availability**: Neon Auth is moving toward general availability with additional plugins and features on the way. Check the [Neon Auth roadmap](/docs/auth/roadmap) for details.
- **Performance optimizations for large computes**: We're working on improving startup performance for larger compute instances after restarts.

> - **Large object storage**: We're working on adding support for large object storage.

Reviewer: Should probably drop this.

Author: drop large object storage?

Author: ```suggestion
```

> - **Blob Storage**: S3-compatible object storage with metadata in your database as the source of truth. Buckets branch alongside your project, so storage state moves with your data.

Reviewer: ```suggestion
- **File Storage**: S3-compatible object storage with metadata automatically synced to your database as the source of truth. Storage branches with your project, so storage state moves with your data.
```

> so storage state moves with your data
I don't like this but I don't know what else to say here.
